graspologic/utils/utils.py

```diff
@@ -249,7 +249,7 @@ def to_laplace(graph, form="DAD", regularizer=None):
     matrix of degrees of each node raised to the -1/2 power, I is the
     identity matrix, and A is the adjacency matrix.
 
-    R-DAD is regularized Laplacian: where :math:`D_t = D + regularizer*I`.
+    R-DAD is regularized Laplacian: where :math:`D_t = D + regularizer \times I`.
```

Comment:
didn't you say that you need to escape backslash in :math: latex?

Reply:
For some reason sphinx will take \times but not \neq

Reply:
it may be that it makes \n into a new line? not sure. but if this is the case - feel free to resolve all relevant conversations.